crane-cloud/frontend
# Chore: admin designs feedback

# Description

Work on admin side feedback, Add admin app footers, round tables  and card re-alignment

## Type of change

## Trello Ticket ID

https://trello.com/c/4rnwpUlV

## How Can This Been Tested?

Test out for a footer on the admin side screens. Check the alignments of the dashboard cards. check for admin table edges

## Checklist:


# Screenshots
<img width="1434" alt="Screenshot 2023-09-27 at 01 54 57" src="https://github.com/crane-cloud/frontend/assets/69305400/a84652ed-a28f-4648-97ab-446060287485">
